lib/srv/sess.go

```diff
@@ -860,6 +997,79 @@ func (s *session) startInteractive(ch ssh.Channel, ctx *ServerContext) error {
 			s.log.Debugf("Failed killing the shell: %v", err)
 		}
 	}()
+
+	return nil
+}
+
+// startInteractive starts a new interactive process (or a shell) in the
+// current session.
+func (s *session) startInteractive(ch ssh.Channel, ctx *ServerContext) error {
+	var err error
```

Comment:
Is this necessary?

I fear you may be unintentionally shadowing this down below.